c/ldv-linux-3.4-simple/43_1a_cilled_true-unreach-call_ok_nondet_linux-43_1a-drivers--platform--x86--topstar-laptop.ko-ldv_main0_sequence_infinite_withcheck_stateful.cil.out.c

```diff
@@ -3568,7 +3569,7 @@ void main(void)
   int tmp___0 ;
   int tmp___1 ;
 
-  struct acpi_device *acpi_device_p1 = ldv_successful_malloc(sizeof(struct acpi_device));
+  struct acpi_device *acpi_device_p1 = malloc(sizeof(struct acpi_device));
```

Comment:
This is wrong!
Please do not replace ldv_successful_malloc by malloc.
1. It does not have checking for ERR_PTR
2. It is more general, because it supports malloc returning zero forbidden in SV-COMP

Reply:
The `ldv_successful_malloc` was undefined, that is why I replaced it (so I would say this is still more "correct" than before). The reason why I used the usual malloc is that from the name "ldv_*successful*_malloc" I would guess that neither of 1. and 2. affect this memory allocation.

Reply:
OK, I understood your reasons.
Let's do it correct without quotes